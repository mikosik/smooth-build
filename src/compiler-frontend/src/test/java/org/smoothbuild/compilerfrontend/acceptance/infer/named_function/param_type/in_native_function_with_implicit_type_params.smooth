@Native("impl")
Int myFunc(x);
