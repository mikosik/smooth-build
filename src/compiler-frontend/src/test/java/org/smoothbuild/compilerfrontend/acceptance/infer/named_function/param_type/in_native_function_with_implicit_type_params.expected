@Native("impl")
Int myFunc<A>(A x);
